ckanext/stadtzh_losdharvest/profiles.py

```diff
@@ -107,8 +106,10 @@ def parse_dataset(self, dataset_dict, dataset_ref):
         if publishers:
             dataset_dict["author"] = publishers[0]
 
-        # Tags
-        dataset_dict["tags"] = self._get_tags(dataset_ref)
+        # Tags, notes and timeRange come from the dataset, referenced
+        # from the view by SCHEMA.isBasedOn
+        dataset_dict["notes"], dataset_dict["tags"], dataset_dict["timeRange"] =\
+            self._get_based_on_fields(dataset_ref)
```

Comment:
I think that looks fine, but is it possible to check if this still works if one or more of the fields are not there/not present? I guess it just won't set them, right?